vars/utils.groovy

```diff
@@ -561,17 +561,27 @@ def perfEval() {
             error("Unable to eval all conv jobs")
         }
 
+        // Verify that evaluation created find_db entries before updating golden
```

Comment:
I've added these fixes for gfx90a stage, given the recent node migration - which likely causes the stage to fail on fresh db runs.